scapy/fields.py

```diff
@@ -1460,7 +1460,10 @@ def i2repr(self, pkt, x):
         elif self.use_nano:
             x = x/1e9
         x = int(x) + self.delta
-        t = time.strftime(self.strf, time.gmtime(x))
+        try:
+            t = time.strftime(self.strf, time.gmtime(x))
+        except:
+            t = "error"
```

Comment:
This might seem unrelated but cause some malformed TLS packets to get badly dissected